e2e/cypress/integration/enterprise/elasticsearch_autocomplete/helpers/index.js

```diff
@@ -60,78 +60,89 @@ module.exports = {
         return {
             ironman: {
                 username: withTimestamp('ironman', reverseTimeStamp),
-                firstName: 'Tony',
-                lastName: 'Stark',
+                password: 'passwd',
+                first_name: 'Tony',
```

Comment:
@saturninoabril I'm not entirely grasping the idea for snake_case vs camelCase usage. Is there a reason we're changing it here?

Reply:
This is just matching the object requirements of server so that we don't need to change casing when using on client. You'll appreciate it when dealing with `cy.apiCreateUser`. We're also making this consistent with other like channels, teams, etc.